jira_agile_metrics/cli.py

```diff
@@ -114,6 +115,8 @@ def get_jira_client(connection):
     password = connection['password']
     http_proxy = connection['http_proxy']
     https_proxy = connection['https_proxy']
+    jira_server_version_check = connection['jira_server_version_check']
+    logger.debug("Cheking JIRA server version: %s", jira_server_version_check)
```

Comment:
Can we make this defaults to true if that was the old behaviour? Can you also remove the debug logging here?

Reply:
The default value is set to true in config.py
I am removing the debug line.